packages/sdk-codegen/src/python.gen.ts

```diff
@@ -499,7 +500,8 @@ ${this.hooks.join('\n')}
         case 'HashType':
           return {
             default: this.nullStr,
-            name: `MutableMapping[str, ${map.name}]`,
+            // TODO fix the invalid Looker API spec formats like MergeQuery's vis_config
+            name: `MutableMapping[str, Any]`, // `MutableMapping[str, ${map.name}]`,
```

Comment:
Is this over-reaching? shouldn't it remain `${map.name}`?

Reply:
Quite possibly. I can go less extreme and only remap string to Any